src/coreclr/src/vm/clrex.h

```diff
@@ -22,14 +22,23 @@ class AssemblySpec;
 class PEFile;
 class PEAssembly;
 
+enum StackTraceElementFlags
+{
+    // Set if this element represents the last frame of the foreign exception stack trace
+    STEF_LAST_FRAME_FROM_FOREIGN_STACK_TRACE = 0x0001,
+
+    // Set if the "ip" field has already been adjusted (decremented)
+    STEF_IP_ADJUSTED = 0x0002,
+};
+
+// This struct is used by SOS in the diagnostic repo and needs to remain backwards compatible.
```

Comment:
We need to be in position to change the runtime internal implementation details as necessary with a matching updates to SOS.

I am fine with keeping things backward compatible to make thing simple, but I do not think that the comments should be saying that this can never ever change.

Reply:
I'll change the comments removing " and needs to remain backwards compatible".